ngraph/core/include/ngraph/function.hpp

```diff
@@ -190,16 +190,17 @@ namespace ngraph
     };
 
     template <>
-    class NGRAPH_API AttributeAdapter<std::shared_ptr<Function>> : public VisitorAdapter
+    class NGRAPH_API AttributeAdapter<std::shared_ptr<Function>>
```

Comment:
Do we still need this `AttributeAdapter`?

Reply:
We still need this AttributeAdapter in order to call on_adapter(string name, ValueAccessor<std::shared_ptr<**Function**>>& adapter). It is used by TI and Loop to acquire body Function during deserialization.